test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -40,6 +40,7 @@ export function setup(opts: minimist.ParsedArgs) {
 			await app.workbench.sqlNotebook.waitForPlaceholderGone();
 
 			const text1: string = 'SEL';
+			await app.workbench.sqlNotebook.clickOnCodeCell();
```

Comment:
Are we no longer able to type right away after adding a new code cell?

Reply:
We can type right away when creating a new code cell or after a single click

Reply:
I think @barbaravaldez was saying that we are able to type right away so we shouldn't need this new step in the smoke test right?

Reply:
No change on the current behavior. I thought this was fixing the smoke test failure but it ended up being another issue